### ISSUE TITLE
Accessibility: Hamburger "menu" missing some properties and states of a disclosure control

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1044]
## Issue
The Hamburger Menu does not follow the established design pattern for a disclosure control

## Expected Behavior
- [WCAG 2.2 AA 4.1.2 Name, Role, Value](https://www.w3.org/TR/wai-aria-1.2/#introstates) requires that programmatically determined states and properties are communicated to screen reader users.
- [WAI - ARIA APG Disclosure Show/Hide Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/) requires that when the content is visible, the element with role `button` has `aria-expanded` set to `true`. When the content area is hidden, it is set to `false`.
Optionally, the element with role `button` has a value specified for `aria-controls` that refers to the element that contains all the content that is shown or hidden.
- Screen reader users expect to hear this information when interacting with disclosure buttons to determine the current state of the control
## Acceptance Criteria
- [x] Add an `aria-expanded` attribute on the `HamburgerButton` which returns `true` when expanded and `false` when not expanded 
- [x] Optionally add an `aria-controls` attribute to the `HamburgerButton` which refernces the `id` of the container for the exposed controls 
- [x] Test with a screen reader JAWS, VoiceOver, NVDA, Orca, Talkback, etc. - ensure the expanded state is announced

_Linked to tracking issue: #456_

**Comment by jason-me** [comment]
Pending closure via PR: https://github.com/stakwork/sphinx-tribes-frontend/pull/1140

**Comment by jason-me** [comment]
Merged in PR: https://github.com/stakwork/sphinx-tribes-frontend/pull/1140
